src/hooks/useCioAutocomplete.ts

```diff
@@ -141,6 +142,7 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
     request,
     featureToggles: features,
     isOpen: isOpen && items?.length > 0,
+    isFocused: isOpen || inputFocusState,
```

Comment:
I believe isOpen is effectively the input focus state but I wasn't sure.

Reply:
I believe yes

Reply:
I believe you added `isFocused` becuase our `isOpen: isOpen && items?.length > 0`

I think we need to change that since the dropdown can be open now even though `items?.length === 0` so it's not accurate anymore

Reply:
If we did that we won't need to add `isFocused`

Reply:
Yea that is why. If it just use isOpen it will solve the edge cases you mentioned in the other comment.

Reply:
But I am not sure if removing items?.length check will affect anything else, if a customer has behavior that relies on this logic 🤔

Reply:
It probably will. Lets discuss with the broader team

I think we can still make the change and mark it breaking maybe